src/types.jl

```diff
@@ -176,10 +180,11 @@ mutable struct Frame
     framecode::FrameCode
     framedata::FrameData
     pc::Int
+    assignment_counter::Int
```

Comment:
Could potentially make this a concrete `Int64` if we worry at all a out overflow (on 32-bit)